### ISSUE TITLE
activate "C:\portabel\miniconda\envs\dev_35" returns an error

### ISSUE THREAD

**Issue by janschulz** [conda/conda#2780]
On windows, I could use `activate "C:\portabel\miniconda\envs\dev_35"` to activate an env. This seems to be not anymore possible as it returns an error:

```
c:\data\external\jupyter\jupyter_environment_kernels (conda_windows)
[dev_35] λ activate "C:\portabel\miniconda\envs\dev_35"
prepending C:\\portabel\\miniconda\\envs\\dev_35 and C:\\portabel\\miniconda\\envs\\dev_35\\Library\\mingw-w64\\bin and C:\\portabel\\miniconda\\envs\\dev_35\\Library\\usr\\bin and C:\\portabel\\miniconda\\envs\\dev_35\\Library\\bin and C:\\portabel\\miniconda\\envs\\dev_35\\Scripts to PATH
Die Umgebungsvariable "C:\portabel\miniconda\envs\dev_35" ist nicht definiert.
```

(The last is "The env variable "..." is not defined")

[It would also be nice if this wouldn't output anything or at least print the right slashes]

**Comment by janschulz** [comment]
It seems it misses a `GOTO :EOF` before the `:TRIM` line

**Comment by msarahan** [comment]
Possibly fixed by #2796 - agree about reducing the printout.  Can we have a little discussion here, @kalefranz ?

**Comment by kalefranz** [comment]
I definitely agree with "at least print the right slashes."

And the length of that message on Windows is incredibly annoying.

I vote "don't print anything." If you want to know what's happening under the covers, there are other ways to find out.

**Comment by msarahan** [comment]
I believe the errors are fixed in the current 4.1.x branch HEAD.  I'm too swamped to debate or change cosmetic issues, but I support any reduction in output that anyone wants to do.

**Comment by kalefranz** [comment]
mostly fixed in #2796

**Comment by janschulz** [comment]
Thanks a lot!
